data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -0,0 +1,45 @@
+from typing import Iterable
+import pandas as pd
+
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from googleapiclient.discovery import Resource
+
+
+# this connection type will be changed !!
+def connect_to_email(userId: str) -> Resource:
```

Comment:
minor: naming, perhaps rename parameter to `user_id`?

Maybe worth also reviewing the function name. But it's always difficult. Perhaps build gmail service, but then it's also getting credentials. Just a thought anyway.

Reply:
I have changed the function name.